### DISCUSSION TITLE
503 Service Temporarily Unavailable [Oct 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6735] [Discord Threads]
> 🙋‍♂️ **.alex.pro**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### Can you share more details like when you see 503 error?  
#  
> 👨‍🚀 **wingkwong**  
#### > copy details   
####    
#### do u mean Project info?  
![image0](https://cdn.discordapp.com/attachments/1150345454446456905/1150407177891229757/image.png?ex=653e4fac&is=652bdaac&hm=ea7655e8d0257d2f6c475a9643fabfeacc5498f6e2afc597f4df076fb163ec7a&)  
#  
> 👨‍💻 **pranavxc**  
#### Are you using nginx or any loadbalancer in between, could be related to that. And if possible can you share the server/docker logs as well  
#  
> 🙋‍♂️ **.alex.pro**  
#### Node: **v16.20.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.111.1**  
#  
> 🙋‍♂️ **.alex.pro**  
#### Logs  are unreadable ... it'd be better to have a call and check online  
#  
> 👨‍💻 **pranavxc**  
#### sure I will share the deatils in DM  
#  
> 🙋‍♂️ **.alex.pro**  
#### fixed! 😄  
#  
> 👨‍💻 **pranavxc**  
#### Issue could be related to - https://github.com/nocodb/nocodb/issues/6345 and it's fixed in latest release. 👍  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1150345454446456905)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
